electives/18847.md

```diff
@@ -0,0 +1,41 @@
+# 18-847C: Data Center Computing
+
+| Category | Difficulty |
+|:-:       | :-:        |
+| Reviews  | 5          |
+| Project  | 6          |
+| Present  | 7          |
+
+This class gives a introduction to data center computing issues by reading recently published research papers addressing current problems that datacenters face today, as well as through a semester long research project into a data center problem. 
+
+## Paper Reviews
+
+Each class (meet twice a week), you are required to read and review two reseach papers. During class, these papers are presented and discussed. Sometimes, one of the reseachers who wrote the paper will visit for discussion. The most important part of the reviews is the critical analysis of where they could have improved and your discussion questions. 
+
+## Paper Presentations
+
+You are required to prepare 2-3 paper presentations throughout the semester. These presentations are 10-15 minutes long and can be relatively informal. Each class, two of these presentations occur, each one focusing on one of the papers for the class. These often take long to prepare, contributing to a higher number of hours spent on the class for those weeks. 
```

Comment:
Why do these presentations take a long time to prepare? What are ways to help people get unstuck while they are making these presentations?

Reply:
It depends on how familiar you are with the material and how good of a presentation you want to make it.

Reply:
See commit https://github.com/gfieni2022/CMU-ECE-CS-Guide/commit/c2eddb433ab1014933531ef80057d0447d8dc86c